.buildkite/scripts/dra_release.sh

```diff
@@ -41,7 +41,9 @@ mkdir -p ${BASE_DIR}/reports
 ./dev-tools/dependencies-report --csv ${BASE_DIR}/reports/dependencies-${VERSION}.csv
 cd ${BASE_DIR}/reports && shasum -a 512 dependencies-${VERSION}.csv > dependencies-${VERSION}.csv.sha512
 
-cd $(dirname ${WORKSPACE})
+cd ${WORKSPACE}
 export FOLDER="${FOLDER_PATH}"
 export OUTPUT_FILE="${DRA_OUTPUT}"
+echo "$(pwd)"
+ls -la
```

Comment:
nit: do we need this?

Reply:
We are currently debugging this pipeline as we are failing to use the dra scripts.
Unfortunately we have to be on main to test it.